### PR TITLE
Updating TV-casting-Android app read me and tv-app linux

### PR DESCRIPTION
**Pull request by ajoletta**
Updating TV-casting-Android read me with environment variable location examples and TV-casting Linux read me with ux enabling command for successful commissioning to the controller

### CODE REVIEW

File: examples/tv-casting-app/android/README.md [hunk 1]
```diff
@@ -21,9 +21,10 @@ the TV.
 ## Requirements for building
 
 You need Android SDK 21 & NDK 23.2.8568313 downloaded to your machine. Set the
```
Comment (sharadb-amazon) [inline examples/tv-casting-app/android/README.md:23]: Can you make a change to omit stating any specific version numbers for the Android SDK or NDK, and instead provide a link to this doc: https://github.com/project-chip/connectedhomeip/blob/7a815f675c15e7b9d35bddde1324306ce7887b9e/docs/guides/android_building.md#requirements-for-building

**Review by sharadb-amazon — CHANGES_REQUESTED** [review]

File: examples/tv-app/linux/README.md [hunk 2]
```diff
@@ -55,6 +55,10 @@ Try the "commission-onnetwork <pincode> <disc> <IP> <port>" command
     $ controller commission-onnetwork 34567890 2976 192.168.65.3 5540 -or-
     $ controller commission-onnetwork 34567890 2976 fe80::50:ff:fe00:1 5540
 
+If you receive a pairing failure, run
+
+    $ controller ux ok
+
```
[the remark this answers is not in the document]
Comment (ajoletta) [reply examples/tv-app/linux/README.md]: Updated

File: examples/tv-casting-app/android/README.md [hunk 3]
```diff
@@ -20,10 +20,17 @@ the TV.
 
 ## Requirements for building
 
-You need Android SDK 21 & NDK 23.2.8568313 downloaded to your machine. Set the
-`$ANDROID_HOME` environment variable to where the SDK is downloaded and the
-`$ANDROID_NDK_HOME` environment variable to point to where the NDK package is
-downloaded.
+Refer to
+[this file](https://github.com/project-chip/connectedhomeip/blob/7a815f675c15e7b9d35bddde1324306ce7887b9e/docs/guides/android_building.md#requirements-for-building)
```
Comment (sharadb-amazon) [inline examples/tv-casting-app/android/README.md:24]: Is there a way to use relative linking?

Comment (ajoletta) [reply examples/tv-casting-app/android/README.md:24]: resolved

**Review by sharadb-amazon — APPROVED** [review]

**Comment by ajoletta** [comment]
@chrisdecenzo Could I please get another approval on my PR?

File: .github/dependabot.yml [hunk 4]
```diff
@@ -1,5 +1,14 @@
 version: 2
 updates:
+  - package-ecosystem: "gradle"
```
Comment (cliffamzn) [inline .github/dependabot.yml:3]: might want to resync here, these changes don't seem right

Comment (ajoletta) [reply .github/dependabot.yml]: Yup, thanks I just fixed it

File: examples/tv-app/linux/README.md [hunk 5]
```diff
@@ -55,6 +55,20 @@ Try the "commission-onnetwork <pincode> <disc> <IP> <port>" command
...
 
+When the tv-app receives a UDC request from a commissionable node (like a
+tv-casting-app), it will print out
+
+    CHIP:SVR: UserDirectedCommissioningServer::OnMessageReceived
+    CHIP:SVR: UDC instance=F879911BF17129AA
+    CHIP:SVR: OnCommissionableNodeFound instance: name=F879911BF17129AA old_state=1 new_state=3
+    CHIP:CTL: ------PROMPT USER: Test TV casting app is requesting permission to cast to this TV, approve? [0x0000_FFF1,0x0000_8001,F879911BF17129AA,01005DD9BB0990AF18F19C35C30C670532BC]
+    CHIP:CTL: ------Via Shell Enter: controller ux ok|cancel
+    CHIP:CTL: Unknown filter type; all matches will fail
```
Comment (chrisdecenzo) [inline examples/tv-app/linux/README.md:66]:
```suggestion
```
I don't think this last line is relevant, right?

**Review by chrisdecenzo — APPROVED** [review]

File: examples/tv-app/linux/README.md (hunk 5 above)
Comment (ajoletta) [reply examples/tv-app/linux/README.md]: I'll remove it

**Review by hawk248 — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**